demo/public/index.html

```diff
@@ -2,9 +2,9 @@
 <html>
 <head>
   <title>VHX Quartz React Demo</title>
-  <link rel="stylesheet" href="http://quartz.vhx.tv/quartz.css">
-  <link rel="stylesheet" href="http://quartz.vhx.tv/quartz-icons.css">
-  <link rel="stylesheet" href="http://quartz.vhx.tv/components.css">
+  <link rel="stylesheet" href="/css/quartz.css">
+  <link rel="stylesheet" href="/css/quartz-icons.css">
+  <link rel="stylesheet" href="/css/components.css">
```

Comment:
I like that this is a locally hosted version of some quartz components (for now).  Perhaps on another ticket we should consider installing Quartz as a package and using it in our .scss files as needed?